str_analysis/call_non_ref_pathogenic_motifs.py

```diff
@@ -391,7 +401,7 @@ def compute_final_expansion_hunter_results(locus_results_json, output_file_prefi
 
         if ("expansion_hunter_motif1_long_allele_size" not in locus_results_json or
             "expansion_hunter_motif2_long_allele_size" not in locus_results_json):
-            # this case can occur when low coverage at this locus prevents ExpansionHunter from outputting a genotype
+            # This case can occur when low coverage at this locus prevents ExpansionHunter from outputting a genotype
```

Comment:
recommenting just in case:
I think I'm still not quite following, why would you always want to set long_allele_motif to motif2?
if expansion_hunter_motif1_long_allele_size isn't in the locus_results_json, then short allele motif is motif 1, and the long allele motif is motif2. but, if expansion_hunter_motif2_long_allele_size isn't in the locus_results_json, then the short allele is still motif1 and the long allele is still motif2. why wouldn't you want to set the results differently based on which long allele is missing?